### ISSUE TITLE
[UI] Provide a way to access the text-only log for a job *directly* from the build dashboard

### ISSUE THREAD

**Issue by ribasushi** [travis-ci/travis-ci#3604]
The Travis-CI UI is heavy... at times way too heavy for loaded workstations. Opening ~10 build logs in tabs in parallel takes upwards of **30** seconds on my non-ancient I7 laptop.

It would be an **absolute boon** to my and others productivity if there is a way to open [a text-only log](https://s3.amazonaws.com/archive.travis-ci.org/jobs/57352573/log.txt) of a [particular job](https://travis-ci.org/dbsrgits/dbix-class/jobs/57352573) directly from [a build page](https://travis-ci.org/dbsrgits/dbix-class/builds/57352571). It doesn't really matter how exactly this is implemented, as long as there is a way to jump to a text log directly from the build page.

Thank you!

/cc @haarg

**Comment by joshk** [comment]
Hi @ribasushi 

Thanks for the feature suggestion.

I'm sorry, but after thinking about this feature, we won't be adding it to the UI at this time.

We would be happy to investigate why logs are taking a log time to load though, but it would be best to do that in a different issue.

Thanks a bundle

Josh

**Comment by kentfredric** [comment]
Not opening a new ticket just yet because I don't have much to go on other than "yah, its feels sluggish" at present, and I'm just starting digging at the most obvious things.

There's some odd slownesses in HTTP alone that look weird there, like 3 seconds to return 204 no content seems odd.

I have caching disabled for this test, and I'm obviously limited by international overheads, but 3 seconds for 2k of data just doesn't sound right.

![sn66](https://cloud.githubusercontent.com/assets/44790/7277471/5eaa7b7e-e964-11e4-9a29-665fcd6c9a65.jpg)
